container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -250,6 +253,22 @@ func CreateEngine(
 			return nil, stacktrace.Propagate(err, "An error occurred waiting for the engine grpc proxy port '%v/%v' to become available", privateGrpcProxyPortSpec.GetTransportProtocol(), privateGrpcProxyPortSpec.GetNumber())
 		}*/
 
+	logrus.Infof("Starting the centralized logs components...")
+	logsCollectorDaemonSet := fluentbit.NewFluentbitLogsCollector()
+
+	_, removeLogsCollectorFunc, err := logs_collector_functions.CreateLogsCollector(ctx, logsCollectorTcpPortNum, logsCollectorHttpPortNum, logsCollectorDaemonSet, nil, kubernetesManager, objAttrsProvider)
```

Comment:
Unlike the `DockerBackend`, where the log collectors are deployed by the engine during enclave creation, in the `k8s` backend, the logs collector lifecycle gets managed with the engines and is created during engine creation.